client.go

```diff
@@ -266,7 +266,7 @@ func (c *HTTPClient) NetworkGetKeyWithStatus(keyID string, status VersionStatus)
 	d.Set("status", string(s))
 
 	key := &Key{}
-	err = c.getHTTPData("GET", "/v0/keys/"+keyID+"/?status="+string(s), nil, key)
+	err = c.getHTTPData("GET", "/v0/keys/"+keyID+"/?status="+string(s), d, key)
```

Comment:
Can you add 

```defer resp.Body.Close()``` in getHTTPData on line 394 right after the client.Do() / error check?

Reply:
I plan to create a separated PR for that, as we have to make another similar change in `server/auth/auth.go`, also I think defer won't work within for loop